src/components/Profile/components/PersonalInfoList/index.js

```diff
@@ -318,8 +469,19 @@ const PersonalInfoList = ({
             {studentID}
             {home}
             {spouse}
+            {disclaimer1}
```

Comment:
In my opinion, the order should be like this...

```suggestion
            {note}
            {disclaimer1}
            {emergencyContact1}
            {emergencyRelationship1}
            {emergencyHomePhone1}
            {emergencyCellPhone1}
            {emergencyWorkPhone1}
            {emergencyContact2}
            {emergencyRelationship2}
            {emergencyHomePhone2}
            {emergencyCellPhone2}
            {emergencyWorkPhone2}
            {disclaimer2}
```